packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Expressions/SystemSnippets.Serializations.JsonFormat.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Generator.CSharp.Expressions;
+
+namespace Microsoft.Generator.CSharp.ClientModel.Expressions
+{
+    internal static partial class SystemSnippets
+    {
+        internal static class SerializationsJsonFormat
+        {
+            internal static StringExpression JsonFormat = Snippets.Literal("J");
```

Comment:
typically this is good to have helpers but in this case its actually more typing to use the helper than just set a literal

`SystemSnippets.SerializationsJsonFormat.JsonFormat`
vs
`Snippets.Literal("J")`

Reply:
ah this was previous feedback from https://github.com/microsoft/typespec/pull/3355#discussion_r1604196243. As a compromise, I'll just keep the `SystemSnippets` class but remove the the nested class. So it can be `SystemSnippets.SerializationJsonFormat`